packages/realm/src/runtime.d.ts

```diff
@@ -42,7 +42,7 @@ declare module "buffer" {
   type Buffer = Uint8Array;
 }
 
-/** @deprecated React Native doesn't provide this */
```

Comment:
You may want to @internal this to ensure it never ends up in our .d.ts file and causes problems for users.

Reply:
Actually, nevermind. That applies to everything in this file, so we must be handling this somehow already.

Reply:
Yeah. Nothing imports the file, it's included via the `tsconfig`:

https://github.com/realm/realm-js/blob/5d00d08103a20e3682b6196419dee46d43daa31b/packages/realm/tsconfig.json#L21

Reply:
But doesn't that get used by rollup when producing our bundle.d.ts?

Reply:
Or I guess rollup uses its own `inputs` rather than the tsconfig's `include`? I could still see this causing issues via ambient inclusion, but if these types aren't showing up in the .d.ts, I guess it is fine. Just something to keep an eye out for.